afkak/consumer.py

```diff
@@ -46,21 +45,21 @@ class Consumer(object):
     This consumer consumes a single partition from a single topic, optionally
     automatically committing offsets.  Use it as follows:
 
-      * Create an instance of :class:`afkak.KafkaClient` with cluster
-        connectivity details.
-      * Create the :class:`Consumer`, supplying the client, topic, partition,
-        processor function, and optionally fetch specifics, a consumer group,
-        and a commit policy.
-      * Call :meth:`.start` with the offset within the partition at which to
-        start consuming messages. See :meth:`.start` for details.
-      * Process the messages in your :attr:`.processor` callback, returning a
-        :class:`~twisted.internet.defer.Deferred` to provide backpressure as
-        needed.
-      * Once processing resolves, :attr:`.processor` will be called again with
-        the next batch of messages.
-      * When desired, call :meth:`.stop` on the :class:`Consumer` to halt
-        calls to the :attr:`processor` function and cancel any outstanding
-        requests to the Kafka cluster.
+    - Create an instance of :class:`afkak.KafkaClient` with cluster
```

Comment:
There are a few places in this PR where I fixed the list syntax by dedenting it like this. The bullets must be in the first column or reStructuredText interprets the list as a blockquote containing a list.